backend/src/main/java/com/woowacourse/ternoko/service/ReservationService.java

```diff
@@ -47,6 +50,11 @@ public Long create(final Long coachId, final ReservationRequest reservationReque
             formItem.addInterview(savedInterview);
         }
 
+        AvailableDateTime availableDateTime = availableDateTimeRepository.findByCoachIdAndInterviewDateTime(
+                        coachId, reservationRequest.getInterviewDatetime())
+                .orElseThrow(() -> new InvalidReservationDateException(ExceptionType.INVALID_AVAILABLE_DATE_TIME));
```

Comment:
개행이 이렇게 되면 깔끔해질 것 같아요
```suggestion
        final AvailableDateTime availableDateTime = availableDateTimeRepository
                .findByCoachIdAndInterviewDateTime(coachId, reservationRequest.getInterviewDatetime())
                .orElseThrow(() -> new InvalidReservationDateException(INVALID_AVAILABLE_DATE_TIME));
```

Reply:
```java
final AvailableDateTime availableDateTime = availableDateTimeRepository
                .findByCoachIdAndInterviewDateTime(coachId, reservationRequest.getInterviewDatetime())
                .orElseThrow(() -> new InvalidReservationDateException(INVALID_AVAILABLE_DATE_TIME));
```
이렇게 수정해보았습니다.